src/NodaTime/Calendars/Era.cs

```diff
@@ -33,6 +33,11 @@ public sealed class Era
         public static Era AnnoMartyrum { get; } = new Era("AM", "Eras_AnnoMartyrum");
 
         /// <summary>
+        /// The "Anno Martyrum" or "Era of the Martyrs". This is the sole era used in the Coptic calendar.
```

Comment:
Sure, I just copied and tweaked a bit, then wanted to commit to ensure that I wasn't wasting time.